### ISSUE TITLE
wallaby imports module under its extension path

### ISSUE THREAD

**Issue by kwonoj** [wallabyjs/public#2451]
### Issue description or question

![image](https://user-images.githubusercontent.com/1210596/84957586-6137c900-b0b0-11ea-9fcc-7b52679a08a9.png)

For some reason, wallaby imports module `uuid` from its extension path instead of project's `node_modules` which fails up test cases due to providing different version than project's pkg.

for other modules in pkg, it pulls properly from pkg's node_modules: 
![image](https://user-images.githubusercontent.com/1210596/84958621-9cd39280-b0b2-11ea-997c-62752f9d2958.png)

I tried to create repro case but failed, somehow new project I created does not have this problem.

//jest.json

```
{
  "preset": "ts-jest/presets/js-with-babel",
  "testResultsProcessor": "<rootDir>/node_modules/jest-junit-reporter",
  "testRegex": "(-spec)\\.(ts|tsx)$",
  "testRunner": "jest-circus/runner",
  "testMatch": null,
  "setupFilesAfterEnv": [
    "<rootDir>/spec/jest/setup.js"
  ],
  "bail": false,
  "resetMocks": false,
  "resetModules": false,
  "globals": {
    "ts-jest": {
      "tsConfig": "spec/jest/tsconfig.json",
      "diagnostics": false,
      "babelConfig": {
        "comments": false
      }
    },
    "__JEST_DEV__": true
  },
  "globalSetup": "<rootDir>/spec/jest/globalSetup.js",
  "moduleFileExtensions": [
    "js",
    "jsx",
    "json",
    "ts",
    "tsx"
  ],
  "testPathIgnorePatterns": [
    "/.tmp/",
    "/spec/mocha",
    "/spec/e2e"
  ],
  "modulePathIgnorePatterns": [
    "/.tmp/"
  ],
  "coverageReporters": [
    "json-summary"
  ],
  "collectCoverageFrom": [
    "src/**/*.{ts, tsx}",
    "!**/*.d.ts"
  ],
  "reporters": [
    "jest-spin-reporter"
  ],
  "coveragePathIgnorePatterns": [
    "<rootDir>/node_modules/",
    "<rootDir>/spec/.*\\.(ts|js)$",
    "<rootDir>/build/.*\\.(ts|js)$"
  ],
  "moduleNameMapper": {
    "\\.(jpg|ico|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/spec/jest/__mocks__/file-mocks.js",
    "@felixrieseberg/nugget$": "<rootDir>/spec/jest/__mocks__/default-stub.js",
    "\/telemetry$": "<rootDir>/spec/jest/__mocks__/telemetry-mock.js",
    "bugsnag-reporter$": "<rootDir>/spec/jest/__mocks__/bugsnag-reporter-mock.js",
    "electron-windows-notifications": "<rootDir>/spec/jest/__mocks__/electron-windows-notifications.js",
    "debug$": "<rootDir>/spec/jest/__mocks__/default-stub.js",
    "@marshallofsound/native-keymap$": "<rootDir>/spec/jest/__mocks__/default-stub.js",
    "lodash-es$": "<rootDir>/node_modules/lodash/index.js",
    "logger$": "<rootDir>/spec/jest/__mocks__/logger-mock.js",
    "macos-notification-state": "<rootDir>/spec/jest/__mocks__/default-stub.js",
    "windows-focus-assist": "<rootDir>/spec/jest/__mocks__/windows-focus-assist.js",
    "^windows-notification-state": "<rootDir>/spec/jest/__mocks__/windows-notification-state.js",
    "windows-quiet-hours": "<rootDir>/spec/jest/__mocks__/windows-quiet-hours.js",
    "\/local-storage$": "<rootDir>/spec/jest/__mocks__/local-storage-mock.js",
    "\/trace-recorder$": "<rootDir>/spec/jest/__mocks__/default-stub.js"
  }
}
```

//wallaby.js

```
module.exports = (wallaby) => ({
  files: [
    'src/**/*.ts?(x)',
    'build/**/*.ts',
    'src/**/*.html',
    'spec/jest/**/*.snap',
    { pattern: 'jest.json', instrument: false, load: true },
    { pattern: 'spec/jest/**/!(*-spec)', instrument: false, load: true },
    { pattern: 'package.json', instrument: false, load: true },

    { pattern: 'node_modules/lodash/**/*.js', instrument: false, load: true },


    { pattern: 'spec/jest/**/*-spec.ts?(x)', ignore: true },

    { pattern: 'node_modules/debug/**/*', ignore: true },
    { pattern: 'node_modules/@marshallofsound/native-keymap/**/*', ignore: true },
  ],

  tests: [ 'spec/jest/**/*-spec.ts?(x)' ],

  env: {
    type: 'node',
    runner: 'node',
    params: { env: 'wallaby=true' }
  },

  trace: true,
  testFramework: 'jest',

  preprocessors: {
    'spec/jest/**/*-spec.js?(x)': (file) =>
      require('@babel/core').transform(
        file.content,
        { sourceMaps: true, filename: file.path, presets: [ require('babel-preset-jest') ] })
  },

  compilers: {
    '**/*.ts?(x)': wallaby.compilers.typeScript({
      jsx: 'React'
    })
  },

  workers: {
    initial: 2,
    regular: 2
  },

  setup: (w) => {
    const path = require('path');

    const jestConfig = (({
      resetMocks,
      resetModules,
      globals,
      moduleNameMapper,
      setupFilesAfterEnv,
      modulePathIgnorePatterns
    }) => ({
      resetMocks,
      resetModules,
      globals,
      moduleNameMapper,
      setupFilesAfterEnv,
      modulePathIgnorePatterns
    }))(require('./jest.json'));

    jestConfig.moduleNameMapper = Object.entries(jestConfig.moduleNameMapper).reduce(
      (acc, [ key, value ]) => {
        acc[key] = path.join(wallaby.projectCacheDir, value.replace(`<rootDir>/`, ''));
        return acc;
      },
      {}
    );

    jestConfig.setupFilesAfterEnv = jestConfig.setupFilesAfterEnv.map((filePath) =>
      path.join(
        wallaby.projectCacheDir,
        filePath.replace(`<rootDir>/`, '')
      ))

    w.testFramework.configure(jestConfig);
  }
});
```

There was no specific mock to `uuid` module itself.

### Wallaby diagnostics report

```
{
  editorVersion: '1.46.0',
  pluginVersion: '1.0.210',
  editorType: 'VSCode',
  osVersion: 'win32 10.0.19041',
  nodeVersion: 'v13.7.0',
  coreVersion: '1.0.910',
  config: {
    files: [
      { pattern: 'src/**/*.ts?(x)', ignore: false, trigger: true, load: true, instrument: true, order: 1 },
      { pattern: 'build/**/*.ts', ignore: false, trigger: true, load: true, instrument: true, order: 2 },
      { pattern: 'src/**/*.html', ignore: false, trigger: true, load: true, instrument: true, order: 3 },
      { pattern: 'spec/jest/**/*.snap', ignore: false, trigger: true, load: true, instrument: true, order: 4 },
      { pattern: 'jest.json', instrument: false, load: true, ignore: false, trigger: true, order: 5 },
      { pattern: 'spec/jest/**/!(*-spec)', instrument: false, load: true, ignore: false, trigger: true, order: 6 },
      { pattern: 'package.json', instrument: false, load: true, ignore: false, trigger: true, order: 7 },
      { pattern: 'node_modules/lodash/**/*.js', instrument: false, load: true, ignore: false, trigger: true, order: 8 },
      { pattern: 'spec/jest/**/*-spec.ts?(x)', ignore: true, trigger: true, load: true, instrument: true },
      { pattern: 'node_modules/debug/**/*', ignore: true, trigger: true, load: true, instrument: true },
      { pattern: 'node_modules/@marshallofsound/native-keymap/**/*', ignore: true, trigger: true, load: true, instrument: true }
    ],
    tests: [ { pattern: 'spec/jest/**/*-spec.ts?(x)', ignore: false, trigger: true, load: true, test: true, order: 9 } ],
    env: { type: 'node', runner: 'node', params: { env: 'wallaby=true' }, viewportSize: { width: 800, height: 600 }, options: { width: 800, height: 600 }, bundle: true },
    trace: true,
    testFramework: { version: 'jest@0.4.3', configurator: 'jest@0.4.3', reporter: 'jest@0.4.3', starter: 'jest@0.4.3' },
    preprocessors: { 'spec/jest/**/*-spec.js?(x)': [Function: spec/jest/**/*-spec.js?(x)], 'package.json': [Function (anonymous)] },
    compilers: { '**/*.?(lit)coffee?(.md)': [Function (anonymous)] },
    workers: { initial: 2, regular: 2, recycle: false },
    diagnostics: {},
    filesWithNoCoverageCalculated: [],
    runAllTestsInAffectedTestFile: false,
    updateNoMoreThanOneSnapshotPerTestFileRun: false,
    maxConsoleMessagesPerTest: 100,
    autoConsoleLog: true,
    delays: { run: 0, edit: 100, update: 0 },
    teardown: undefined,
    hints: {
      ignoreCoverage: '__REGEXP /ignore coverage|istanbul ignore/',
      ignoreCoverageForFile: '__REGEXP /ignore file coverage/',
      commentAutoLog: '?',
      testFileSelection: { include: '__REGEXP /file\\.only/', exclude: '__REGEXP /file\\.skip/' }
    },
    automaticTestFileSelection: true,
    runSelectedTestsOnly: false,
    extensions: {},
    reportUnhandledPromises: true,
    slowTestThreshold: 75,
    lowCoverageThreshold: 80,
    loose: true,
    configCode: 'module.exports = (wallaby) => ({\n' +
      '  files: [\n' +
      "    'src/**/*.ts?(x)',\n" +
      "    'build/**/*.ts',\n" +
      "    'src/**/*.html',\n" +
      "    'spec/jest/**/*.snap',\n" +
      "    { pattern: 'jest.json', instrument: false, load: true },\n" +
      "    { pattern: 'spec/jest/**/!(*-spec)', instrument: false, load: true },\n" +
      "    { pattern: 'package.json', instrument: false, load: true },\n" +
      '\n' +
      '    //lodash-es is mapped to lodash via modulenamemapper, load explicitly\n' +
      "    { pattern: 'node_modules/lodash/**/*.js', instrument: false, load: true },\n" +
      '\n' +
      '    //do not load test cases\n' +
      "    { pattern: 'spec/jest/**/*-spec.ts?(x)', ignore: true },\n" +
      '\n' +
      '    //stubbed by modulenamemapper, do not load\n' +
      "    { pattern: 'node_modules/debug/**/*', ignore: true },\n" +
      "    { pattern: 'node_modules/@marshallofsound/native-keymap/**/*', ignore: true },\n" +
      '  ],\n' +
      '\n' +
      "  tests: [ 'spec/jest/**/*-spec.ts?(x)' ],\n" +
      '\n' +
      '  env: {\n' +
      "    type: 'node',\n" +
      "    runner: 'node',\n" +
      "    //this'll allow test can determine runner environment\n" +
      "    params: { env: 'wallaby=true' }\n" +
      '  },\n' +
      '\n' +
      '  trace: true,\n' +
      "  testFramework: 'jest',\n" +
      '\n' +
      '  // Enable mock hoisting as same as ts-jest does\n' +
      '  // (https://github.com/kulshekhar/ts-jest#supports-automatic-of-jestmock-calls)\n' +
      '  preprocessors: {\n' +
      "    'spec/jest/**/*-spec.js?(x)': (file) =>\n" +
      "      require('@babel/core').transform(\n" +
      '        file.content,\n' +
      "        { sourceMaps: true, filename: file.path, presets: [ require('babel-preset-jest') ] })\n" +
      '  },\n' +
      '\n' +
      '  compilers: {\n' +
      "    '**/*.ts?(x)': wallaby.compilers.typeScript({\n" +
      "      jsx: 'React'\n" +
      '    })\n' +
      '  },\n' +
      '\n' +
      '  workers: {\n' +
      '    initial: 2,\n' +
      '    regular: 2\n' +
      '  },\n' +
      '\n' +
      '  setup: (w) => {\n' +
      "    const path = require('path');\n" +
      '\n' +
      '    const jestConfig = (({\n' +
      '      resetMocks,\n' +
      '      resetModules,\n' +
      '      globals,\n' +
      '      moduleNameMapper,\n' +
      '      setupFilesAfterEnv,\n' +
      '      modulePathIgnorePatterns\n' +
      '    }) => ({\n' +
      '      resetMocks,\n' +
      '      resetModules,\n' +
      '      globals,\n' +
      '      moduleNameMapper,\n' +
      '      setupFilesAfterEnv,\n' +
      '      modulePathIgnorePatterns\n' +
      "    }))(require('./jest.json'));\n" +
      '\n' +
      '    jestConfig.moduleNameMapper = Object.entries(jestConfig.moduleNameMapper).reduce(\n' +
      '      (acc, [ key, value ]) => {\n' +
      "        acc[key] = path.join(wallaby.projectCacheDir, value.replace(`<rootDir>/`, ''));\n" +
      '        return acc;\n' +
      '      },\n' +
      '      {}\n' +
      '    );\n' +
      '\n' +
      '    jestConfig.setupFilesAfterEnv = jestConfig.setupFilesAfterEnv.map((filePath) =>\n' +
      '      path.join(\n' +
      '        wallaby.projectCacheDir,\n' +
      "        filePath.replace(`<rootDir>/`, '')\n" +
      '      ))\n' +
      '\n' +
      '    w.testFramework.configure(jestConfig);\n' +
      '  }\n' +
      '});\n'
  },
  packageJSON: {
    dependencies: {
      '@bugsnag/js': '7.1.1',
      '@bugsnag/plugin-react': '7.1.1',
      '@marshallofsound/native-keymap': '2.2.1',
      '@webcomponents/custom-elements': '1.4.0',
      '@webcomponents/webcomponentsjs': '2.4.2',
      bonjour: '3.5.0',
      chokidar: '3.4.0',
      classnames: '2.2.5',
      'cli-table-redemption': '1.0.1',
      color: '2.0.1',
      'command-line-args': '5.1.1',
      'command-line-usage': '6.1.0',
      connect: '3.6.0',
      'electron-redux': '1.5.2',
      'electron-windows-notifications': '3.0.6',
      'electron-winstore-auto-launch': '2.0.3',
      'error-stack-parser': '2.0.6',
      'flux-standard-action': '2.0.3',
      'form-data': '^2.5.0',
      'fs-extra': '4.0.1',
      getos: '3.2.1',
      hammerjs: '2.0.8',
      jszip: '3.2.2',
      'lit-element': '2.2.1',
      'lit-html': '^1.0.0',
      'lodash-es': '4.17.15',
      'macos-notification-state': '1.3.1',
      'message-format': '6.2.3',
      'nothing-mock': '1.0.2',
      react: '16.13.1',
      'react-dom': '16.13.1',
      'react-redux': '7.2.0',
      'react-string-replace-recursively': '0.1.27',
      'react-transition-group': '1.2.1',
      redux: '4.0.5',
      'redux-act': '1.8.0',
      'redux-observable': '1.2.0',
      'redux-persist': '6.0.0',
      'redux-persist-node-storage': '2.0.0',
      'redux-persist-transform-encrypt': '2.0.1',
      reselect: '4.0.0',
      rxjs: '7.0.0-beta.0',
      'rxjs-spy': '7.5.1',
      'sanitize-xml-string': '1.1.0',
      semver: '7.1.3',
      signals: '1.0.0',
      temp: '0.9.1',
      'url-parse': '1.4.7',
      uuid: '7.0.1',
      'windows-active-process': '1.0.1',
      'windows-focus-assist': '1.2.1',
      'windows-notification-state': '1.3.4',
      'windows-quiet-hours': '2.0.0',
      winston: '3.2.1',
      'write-file-atomic': '3.0.3',
      'xml-escape': '1.1.0'
    },
    devDependencies: {
      '@bazel/bazelisk': '1.3.0',
      '@bundle-analyzer/core': '0.5.1',
      '@felixrieseberg/nugget': '2.1.1',
      '@paulcbetts/linux-distro': '3.0.2',
      '@types/bonjour': '3.5.6',
      '@types/classnames': '0.0.32',
      '@types/color': '2.0.1',
      '@types/command-line-args': '5.0.0',
      '@types/command-line-usage': '5.0.1',
      '@types/commander': '2.9.1',
      '@types/connect': '3.4.30',
      '@types/copy-webpack-plugin': '5.0.0',
      '@types/electron-packager': '14.0.0',
      '@types/eslint': '6.8.0',
      '@types/fs-extra': '4.0.0',
      '@types/hammerjs': '2.0.34',
      '@types/java': '0.7.32',
      '@types/jest': '25.2.3',
      '@types/jszip': '3.1.6',
      '@types/lodash': '4.14.149',
      '@types/lodash-es': '4.17.3',
      '@types/node': '12.7.1',
      '@types/node-fetch': '1.6.8',
      '@types/prettier': '2.0.0',
      '@types/progress': '2.0.3',
      '@types/react': '16.9.36',
      '@types/react-dom': '16.9.8',
      '@types/react-redux': '7.1.9',
      '@types/react-transition-group': '1.1.3',
      '@types/semver': '5.5.0',
      '@types/shelljs': '0.8.7',
      '@types/temp': '0.8.34',
      '@types/terser-webpack-plugin': '3.0.0',
      '@types/uuid': '7.0.0',
      '@types/webpack': '4.41.7',
      '@types/webpack-env': '1.15.1',
      '@types/webpack-sources': '0.1.6',
      '@typescript-eslint/eslint-plugin': '2.22.0',
      '@typescript-eslint/eslint-plugin-tslint': '2.22.0',
      '@typescript-eslint/parser': '2.22.0',
      'add-asset-html-webpack-plugin': '3.1.3',
      asar: '0.14.3',
      'aws-sdk': '2.178.0',
      chalk: '2.3.0',
      commander: '2.9.0',
      'conventional-commits-parser': '3.0.8',
      'copy-webpack-plugin': '5.1.1',
      'cross-env': '7.0.0',
      'css-loader': '3.4.2',
      danger: '9.2.10',
      'date-fns': '1.30.1',
      debug: '2.6.9',
      deepmerge: '4.2.2',
      'dependency-cruiser': '8.0.0',
      electron: '9.0.4',
      'electron-notarize': '0.2.0',
      'electron-osx-sign': '0.4.16',
      'electron-packager': '14.1.1',
      'electron-windows-store': '2.0.0',
      'electron-winstaller': '2.5.1',
      'electron-wix-msi': '2.2.0',
      eslint: '6.8.0',
      'eslint-config-typescript': '3.0.0',
      'eslint-plugin-simple-import-sort': '5.0.1',
      'file-loader': '4.2.0',
      'generate-asset-webpack-plugin': '0.3.0',
      github: '12.0.2',
      glob: '7.1.6',
      'hard-source-webpack-plugin': '0.13.1',
      'html-webpack-plugin': '4.0.3',
      husky: '3.0.9',
      jest: '26.0.1',
      'jest-circus': '26.0.1',
      'jest-diff': '26.0.1',
      'jest-extended': '0.11.5',
      'jest-fetch-mock': '1.3.3',
      'jest-junit-reporter': '1.1.0',
      'jest-leak-fixer': '0.0.5',
      'jest-spin-reporter': '2.0.0',
      'jest-worker': '26.0.0',
      'json-stable-stringify': '1.0.1',
      'legal-eagle': '0.16.0',
      less: '3.11.1',
      'less-loader': '5.0.0',
      'lint-staged': '10.0.8',
      lodash: '4.17.15',
      'mini-css-extract-plugin': '0.8.0',
      'node-fetch': '1.7.1',
      'node-native-loader': 'github:mnquintana/node-native-loader#3c0e806',
      'npm-config-arguments': '1.1.3',
      'npm-run-all': '4.1.5',
      'patch-package': '6.2.1',
      'postinstall-postinstall': '2.0.0',
      prettier: '1.19.1',
      progress: '2.0.3',
      'raw-loader': '3.1.0',
      'rx-sandbox': '2.0.0-beta.1',
      shelljs: '0.8.4',
      shx: '0.3.2',
      'spawn-rx': '3.0.0',
      'term-img': '4.1.0',
      'terser-webpack-plugin': '3.0.3',
      'ts-jest': '26.1.0',
      'ts-loader': '7.0.2',
      'ts-node': '8.10.1',
      tslint: '5.18.0',
      'tslint-eslint-rules': '4.1.1',
      'tslint-microsoft-contrib': '5.0.1',
      'tslint-react': '3.2.0',
      tsutils: '2.13.0',
      typedoc: '0.10.0',
      'typedoc-plugin-external-module-name': '1.0.10',
      'typedoc-plugin-internal-external': '1.0.10',
      'typedoc-plugin-ui-router': '1.0.6',
      typescript: '3.9.5',
      'unzip-crx-3': '0.2.0',
      'url-loader': '2.2.0',
      webpack: '4.43.0',
      'webpack-cli': '3.3.11',
      'webpack-merge': '4.2.2',
      webpackbar: '4.0.0'
    }
  },
  fs: { numberOfFiles: 1658 },
  debug: [
    '2020-06-17T22:42:06.898Z project Wallaby Node version: v13.7.0\n',
    '2020-06-17T22:42:06.899Z project Wallaby config: C:\\github\\desktop\\wallaby.js\n',
    '2020-06-17T22:42:09.821Z project File cache: C:\\Users\\okwon\\.vscode\\extensions\\wallabyjs.wallaby-vscode-1.0.210\\projects\\0efcf3c026a284c0\n',
    '2020-06-17T22:42:09.861Z uiService Listening port 51235\n',
    '2020-06-17T22:42:10.003Z workers Parallelism for initial run: 2, for regular run: 2\n',
    '2020-06-17T22:42:10.003Z workers Starting run worker instance #0\n',
    '2020-06-17T22:42:10.003Z workers Starting run worker instance #1\n',
    '2020-06-17T22:42:10.056Z workers Web server is listening at 51429\n',
    '2020-06-17T22:42:10.906Z project File cache requires some updates, waiting required files from IDE\n',
    '2020-06-17T22:42:10.930Z workers Started run worker instance (delayed) #0\n',
    '2020-06-17T22:42:10.930Z workers Started run worker instance (delayed) #1\n',
    '2020-06-17T22:42:12.097Z project Stopping process pool\n',
    '2020-06-17T22:42:12.100Z project Running postprocessor\n',
    '2020-06-17T22:42:12.185Z postprocessor New TypeScript language service is required\n',
    '2020-06-17T22:42:29.828Z project Re-attaching document block comment\n',
    '2020-06-17T22:42:30.104Z project Re-attaching document block comment\n',
    '2020-06-17T22:42:30.120Z project Re-attaching document block comment\n',
    '2020-06-17T22:42:30.842Z project Re-attaching document block comment\n',
    '2020-06-17T22:42:30.906Z project Re-attaching document block comment\n',
    '2020-06-17T22:42:30.934Z project Re-attaching document block comment\n',
    '2020-06-17T22:42:30.949Z project Re-attaching document block comment\n',
    '2020-06-17T22:42:31.103Z project Re-attaching document block comment\n',
    '2020-06-17T22:42:31.173Z project Re-attaching document block comment\n',
    '2020-06-17T22:42:31.200Z project Re-attaching document block comment\n',
    '2020-06-17T22:42:31.203Z project Re-attaching document block comment\n',
    '2020-06-17T22:42:31.206Z project Re-attaching document block comment\n',
    '2020-06-17T22:42:31.206Z project Re-attaching document block comment\n',
    '2020-06-17T22:42:31.208Z project Re-attaching document block comment\n',
    '2020-06-17T22:42:31.208Z project Re-attaching document block comment\n',
    '2020-06-17T22:42:31.210Z project Re-attaching document block comment\n',
    '2020-06-17T22:42:31.307Z project Re-attaching document block comment\n',
    '2020-06-17T22:42:31.310Z project Re-attaching document block comment\n',
    '2020-06-17T22:42:31.320Z project Re-attaching document block comment\n',
    '2020-06-17T22:42:31.322Z project Re-attaching document block comment\n',
    '2020-06-17T22:42:31.458Z project Re-attaching document block comment\n',
    '2020-06-17T22:42:31.458Z project Re-attaching document block comment\n',
    '2020-06-17T22:42:31.463Z project Re-attaching document block comment\n',
    '2020-06-17T22:42:31.494Z project Re-attaching document block comment\n',
    '2020-06-17T22:42:31.496Z project Re-attaching document block comment\n',
    '2020-06-17T22:42:31.498Z project Re-attaching document block comment\n',
    '2020-06-17T22:42:31.586Z project Re-attaching document block comment\n',
    '2020-06-17T22:42:31.593Z project Re-attaching document block comment\n',
    '2020-06-17T22:42:31.651Z project Re-attaching document block comment\n',
    '2020-06-17T22:42:32.018Z project Re-attaching document block comment\n',
    '2020-06-17T22:42:32.025Z project Re-attaching document block comment\n',
    '2020-06-17T22:42:32.152Z project Re-attaching document block comment\n',
    '2020-06-17T22:42:32.223Z project Re-attaching document block comment\n',
    '2020-06-17T22:42:32.226Z project Re-attaching document block comment\n',
    '2020-06-17T22:42:32.867Z project Re-attaching document block comment\n',
    '2020-06-17T22:42:38.306Z project Postprocessor execution finished\n',
    '2020-06-17T22:42:38.306Z project Test run started; run priority: 3\n',
    '2020-06-17T22:42:38.314Z project Running all tests\n',
    '2020-06-17T22:42:38.356Z workers Starting test run, priority: 3\n',
    '2020-06-17T22:42:38.356Z nodeRunner Starting sandbox [worker #0, session #yop13]\n',
    '2020-06-17T22:42:38.357Z nodeRunner Preparing sandbox [worker #0, session #yop13]\n',
    '2020-06-17T22:42:38.357Z nodeRunner Prepared sandbox [worker #0, session #yop13]\n',
    '2020-06-17T22:42:38.358Z workers [worker #0, session #yop13] Running tests in sandbox\n',
    '2020-06-17T22:42:44.638Z workers [yop13] Loaded unknown number of test(s)\n',
    '2020-06-17T22:42:44.652Z workers [yop13] Test executed: creates a new session id on first call\n',
    '2020-06-17T22:42:44.952Z workers [yop13] Run 1 test(s), skipped 8 test(s)\n',
    '2020-06-17T22:42:44.959Z workers [yop13] Sandbox is responsive, closing it\n',
    '2020-06-17T22:42:44.964Z project Test run finished\n',
    '2020-06-17T22:42:44.968Z project Processed console.log entries\n',
    '2020-06-17T22:42:44.972Z project Processed loading sequences\n',
    '2020-06-17T22:42:44.978Z project Processed executed tests\n',
    '2020-06-17T22:42:45.314Z project Processed code coverage\n',
    '2020-06-17T22:42:46.177Z project Test run result processed and sent to IDE\n'
  ]
}
```

**Comment by kwonoj** [comment]
current package.json's `uuid` version is 7.0.1 which have export to `v5`, FYI. 

I deleted & retry whole extension cache / project's node_modules both but didn't work still.

**Comment by smcenlly** [comment]
Could you please try updating your project to use our [automatic configuration feature](https://wallabyjs.com/docs/intro/config.html#automatic-configuration) for Jest v24+? Automatic configuration uses the Jest runtime instead of Wallaby runtime so this may fix your problem.

In the meantime, we'll start reviewing your logs.

**Comment by smcenlly** [comment]
We've identified the cause of your problem and this has been fixed in Wallaby core v1.0.911. The issue was specific to jest and did not affect jest when using automatic configuration.

Wallaby should automatically update for you. If not, you may force an immediate update using the VS Code command, `Wallaby.js: Force Core Update`.

Unless you have a good reason to keep your `wallaby.js` configuration file, we also recommend switching to using Automatic Configuration. This should be as simple as:
1. Rename `jest.json` to `jest.config.js` and add `module.exports = ` at the start of the file
2. Delete your `wallaby.js` file
3. In VS Code, run the command `Wallaby.js: Selection Configuration` and select `Automatic Configuration <project directory>`.
